submissions/kikinovk/JS_DOM_API/js/app.js

```diff
@@ -0,0 +1,93 @@
+    image: './image/green_tea.jpg'
+  },{
+    id: '03',
+    title : 'Oolong tea',
+    caption: 'Oolong tea',
+    description: 'Oolong tea is semi-oxidized, so the leaf is allowed to sit for maybe 2-4 hours, before being heated up to halt oxidization. The amount of oxidation affects the flavour and appearance of the tea. Longer oxidization results in a darker oolong which is more similar in taste to a black tea, while shorter oxidization makes it more similar in nature to green tea. When steeped, Oolong tea produces golden or light brown tea with a very delicate flavour resembling neither black nor green tea.',
+    image: './image/oolong_tea.jpg'
+  },{
+    id: '04',
+    title : 'White tea',
+    caption: 'White tea',
+    description: 'White tea is the least processed of all teas. Only the unopened buds and young leaves covered in silver fuzz are used, and they are merely withered and dried. White tea produces a very light coloured infusion with mild flavour. Its caffeine content is even lower than that of green tea and is considered to have a very high level of antioxidants. White tea is best consumed without any additives at all.',
+    image: './image/white tea.jpg'
+  },{
+    id: '05',
+    title : 'Pu-erh tea',
+    caption: 'Pu-erh tea',
+    description: 'Pu-erh tea is a special type of tea that comes from the Yunan province of China and is known for its earthy flavour. It is made out of tea plucked from wild tea trees rather than cultivated bushes and the leaves go through microbial fermentation by pressing the raw leaves together and then storing them for maturity. Pu-erh tea can be either black or green depending on the level of oxidation allowed in the process. Although we don\'t stock any Pu-erh tea on its own, our Coffee Truffle tea has Pu-erh tea as the base onto which we have added coffee beans and cocoa pieces to give it a rich coffee chocolate flavour.',
+    image: './image/pu_erh-tea.jpg'
+  }
+];
+
+let menuList = document.querySelector('.nav__menu');
+let mainSection = document.querySelector('.main');
+
+const createItemMenu = (id, nameItem) => `<button type="button" id="${id}" class="menu__button">${nameItem}</button>`;
+
+const showArticle = (item) => {
+  const createArticle = (item) => `<h1>${item.caption}</h1><img src="${item.image}" alt="${item.caption}"><p>${item.description}</p>`;
+  const newArticle = document.createElement('article');    
+  newArticle.classList.add('main__article');
+  newArticle.innerHTML = createArticle(item);
+  mainSection.appendChild(newArticle);
+};
+
+document.addEventListener('DOMContentLoaded', () => {
+
+  /* create menu */
```

Comment:
You can create a separate function for generating this content. Moreover, you can get rid of these comments. There are redundant ones here. If you create clear names for your variables it'll increase the readability of your code and you'll not need commenting it.